src/test/suite/fixtures/recorded/selectionTypes/clearHeadCoreCurly.yml

```diff
@@ -23,12 +23,13 @@ initialState:
   marks: {}
 finalState:
   documentContents: |-
```

Comment:
do we have a test for "chuck head core curly"?  If not, worth adding I think

Reply:
It will remove the same range since we are only looking at the content ranges in head/tail stage.

Reply:
Hmm that is a bit unfortunate. I'm not entirely sure what is the right thing to do in that situation. I'm not sure there's anything else we can do reliably, but I was kind of expecting it to include the white space

Reply:
In that case we probably need to pass removal and content range to the head/tail stage. Or pass the leading and trailing delimiters or something like that.

Reply:
Yeah I'm just not sure there's a reliable way to do it without all targets needing to know something about head and tail. Because targets need to be able to be smart about constructing removal range, and if head is just taking it and slicing it in half, that could violate the logic, eg we could possibly end up merging tokens

I don't think there's a simple way to do better than what you have here. Unless you have an idea?

Reply:
Unfortunately I don't have any good answers right now

Reply:
No worries I'm happy to ship it and see if we find ourselves wanting this. Head tail does return a token target so at least tries to do some cleanup. Let's see if that's good enough